src/Compilers/VisualBasic/Test/Emit/Attributes/EmitTestStrongNameProvider.vb

```diff
@@ -20,7 +20,7 @@ Partial Public Class InternalsVisibleToAndStrongNameTests
     Public Sub BadInputStream()
         SigningTestHelpers.InstallKey()
         Dim thrownException = New IOException("This is a test IOException")
-        Dim testFileSystem = New TestStrongNameFileSystem() With {
+        Dim testFileSystem = New TestStrongNameFileSystem(Temp.CreateDirectory().Path) With {
```

Comment:
no need to dispose of the result of CreateDirectory, right?

Reply:
Correct. `Temp` handles all the directories / paths created under it.